wsd/DocumentBroker.cpp

```diff
@@ -149,13 +149,14 @@ DocumentBroker::DocumentBroker(ChildType type, const std::string& uri, const Poc
     , _wopiDownloadDuration(0)
     , _mobileAppDocId(mobileAppDocId)
     , _alwaysSaveOnExit(COOLWSD::getConfigValue<bool>("per_document.always_save_on_exit", false))
-#ifdef ENABLE_DEBUG
-    , _unitWsd(UnitWSD::get())
-#endif
+    , _unitWsd(nullptr)
 {
     assert(!_docKey.empty());
     assert(!COOLWSD::ChildRoot.empty());
 
+    if (UnitWSD::isUnitTesting())
```

Comment:
Can we please do this in the initialization list, and have a const pointer?
```
, _unitWsd(UnitWSD::isUnitTesting() ? nullptr : &UnitWSD::get())
```

(Same elsewhere.)

Reply:
Done